auto/summary

```diff
@@ -26,6 +26,8 @@ Unit configuration summary:
   Unix domain sockets support: $NXT_UNIX_DOMAIN
   TLS support: ............... $NXT_OPENSSL
 
+  process isolation: ......... $NXT_ISOLATION
```

Comment:
These summary flags primary needed to summarize `./configure` options, while in this case it's not related to any option and duplicates the output of configure tests.

Actually, I'm ok to keep it because in this particular case it affects configuration, but wouldn't it be more useful if NXT_ISOLATION will also contain a list of supported isolation options?

Reply:
I added this to summary because isolation depends on a bunch of obscure flags and syscalls.
I just did your suggestion and also added the flags detected to the summary info.